### PR TITLE
EWPP-1540: Add media field to Event.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: modules/oe_content_event/config/install/field.field.node.oe_event.oe_event_media.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  handler_settings:
+    target_bundles:
+      av_portal_photo: av_portal_photo
+      av_portal_video: av_portal_video
+      image: image
+    sort:
+      field: _none
+      direction: ASC
+    auto_create: false
+    auto_create_bundle: av_portal_photo
```
Comment (sergepavle) [inline modules/oe_content_event/config/install/field.field.node.oe_event.oe_event_media.yml]:
```suggestion
    auto_create_bundle: ''
```

File: modules/oe_content_event/config/install/field.storage.node.oe_event_media.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+id: node.oe_event_media
+field_name: oe_event_media
+entity_type: node
+type: entity_reference
+settings:
+  target_type: media
+module: core
+locked: false
+cardinality: -1
+translatable: true
```
Comment (sergepavle) [inline modules/oe_content_event/config/install/field.storage.node.oe_event_media.yml]: As I understand according to requirements this field shouldn't be translatable.

File: modules/oe_content_event/config/post_updates/20001_media_field/field.field.node.oe_event.oe_event_media.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  handler_settings:
+    target_bundles:
+      av_portal_photo: av_portal_photo
+      av_portal_video: av_portal_video
+      image: image
+    sort:
+      field: _none
+      direction: ASC
+    auto_create: false
+    auto_create_bundle: av_portal_photo
```
Comment (sergepavle) [inline modules/oe_content_event/config/post_updates/20001_media_field/field.field.node.oe_event.oe_event_media.yml]:
```suggestion
    auto_create_bundle: ''
```

File: modules/oe_content_event/config/post_updates/20001_media_field/field.storage.node.oe_event_media.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+id: node.oe_event_media
+field_name: oe_event_media
+entity_type: node
+type: entity_reference
+settings:
+  target_type: media
+module: core
+locked: false
+cardinality: -1
+translatable: true
```
Comment (sergepavle) [inline modules/oe_content_event/config/post_updates/20001_media_field/field.storage.node.oe_event_media.yml]: As I understand it shouldn't be translatable.

File: tests/features/event.feature [hunk 5]
```diff
@@ -99,6 +99,8 @@ Feature: Event content creation
     Then I should see the text "Organiser name"
     And the "Internal organiser field" is not visible
 
+    And I should see the text "Media"
```
Comment (sergepavle) [inline tests/features/event.feature]: This step looks like doesn't have big sense because on the page we have many strings like that.

**Review by sergepavle — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**